src/modules/configuration/index.ts

```diff
@@ -44,34 +49,49 @@ export const configValues = () => ({
       288: process.env.WEB3_NODE_URL_288,
       137: process.env.WEB3_NODE_URL_137,
       42161: process.env.WEB3_NODE_URL_42161,
-      // 42: process.env.WEB3_NODE_URL_42,
-      // 421611: process.env.WEB3_NODE_URL_421611,
-      // 69: process.env.WEB3_NODE_URL_69,
-      // 4: process.env.WEB3_NODE_URL_4,
-      // 80001: process.env.WEB3_NODE_URL_80001,
       5: process.env.WEB3_NODE_URL_5,
     },
     spokePoolContracts: {
-      [ChainIds.mainnet]: {
-        address: "0x4D9079Bb4165aeb4084c526a32695dCfd2F77381",
-        startBlockNumber: 14819486,
-      },
-      [ChainIds.optimism]: {
-        address: "0xa420b2d1c0841415A695b81E5B867BCD07Dff8C9",
-        startBlockNumber: 8747136,
-      },
-      [ChainIds.arbitrum]: {
-        address: "0xB88690461dDbaB6f04Dfad7df66B7725942FEb9C",
-        startBlockNumber: 12741972,
-      },
-      [ChainIds.boba]: {
-        address: "0xBbc6009fEfFc27ce705322832Cb2068F8C1e0A58",
-        startBlockNumber: 619993,
-      },
-      [ChainIds.polygon]: {
-        address: "0x69B5c72837769eF1e7C164Abc6515DcFf217F920",
-        startBlockNumber: 28604263,
-      },
+      [ChainIds.mainnet]: [
+        {
+          address: "0x4D9079Bb4165aeb4084c526a32695dCfd2F77381",
+          startBlockNumber: 14819486,
+          abi: JSON.stringify(EthereumSpokePool2Abi),
+          acrossVersion: "2",
+        },
+      ],
```

Comment:
Modify the config so as multiple spoke pool contracts can be supported for each chain